projects/samples/robotbenchmark/viewpoint_control/plugins/robot_windows/viewpoint_control/viewpoint_control.html

```diff
@@ -2,45 +2,52 @@
 <html>
   <head>
     <meta charset='UTF-8'>
+    <link type="text/css" rel="stylesheet" href='../../../../include/tabs.css' />
   </head>
   <body>
-    <div id='infotabs' class='webotsTabs webotsRobotWindowTabs'>
-      <ul>
-        <li><a href='#infotab-overview'>Overview</a></li>
-        <li><a href='#infotab-metrics'>Metrics</a></li>
-        <li><a href='#infotab-instructions'>Instructions</a></li>
-      </ul>
-      <div id='infotab-overview'>
-        <p>This benchmark will teach you how to move the 3D viewpoint using the mouse.</p>
-        <p>The tab entitled <a class='tablink' onclick='$("#infotabs").tabs({active:1});'>Metrics</a> measures your progress in an exercise where
-          you have to place the viewpoint at a specific position and orientation. It also allows you to record your performance in
-          the robotbenchmark database (if you are logged-in).</p>
-        <p>The tab entitled <a class='tablink' onclick='$("#infotabs").tabs({active:2});'>Instructions</a> explains you how to use the mouse to
-          control the viewpoint.</p>
+    <div id='infotabs' class='wrapper webotsTabs webotsRobotWindowTabs'>
+      <input class="radio" id="one" name="group" type="radio" checked>
+      <input class="radio" id="two" name="group" type="radio">
+      <input class="radio" id="three" name="group" type="radio">
+      <div class="tabs">
+        <label class="tab" id="one-tab" for="one">Overview</label>
+        <label class="tab" id="two-tab" for="two">Metrics</label>
+        <label class="tab" id="three-tab" for="three">Instructions</label>
       </div>
-      <div id='infotab-metrics'>
-        <div style='text-align:center' title='Your performance increases as you get closer to the target viewpoint position and orientation.'>
-          <span id='achievement'>%</span></div>
-        <div style='text-align:center'>
-          <button id='record-button' disabled onclick='recordPerformance()'>Record</button>
+
+      <div class="panels">
+        <div class="panel" id="one-panel">
+          <p>This benchmark will teach you how to move the 3D viewpoint using the mouse.</p>
+          <p>The tab entitled <a class='tablink' onclick='$("#infotabs").tabs({active:1});'>Metrics</a> measures your progress in an exercise where
+            you have to place the viewpoint at a specific position and orientation. It also allows you to record your performance in
+            the robotbenchmark database (if you are logged-in).</p>
+          <p>The tab entitled <a class='tablink' onclick='$("#infotabs").tabs({active:2});'>Instructions</a> explains you how to use the mouse to
```

Comment:
These links to other tabs in the same document are no longer working.
They should be fixed or removed.

Same on  line 21 an 39.